dexbot/strategies/base.py

```diff
@@ -444,8 +444,10 @@ def cancel_orders(self, orders, batch_only=False):
         if not success and len(orders) > 1 and not batch_only:
             # One of the order cancels failed, cancel the orders one by one
             for order in orders:
-                self._cancel_orders(order)
-        return True
+                success = self._cancel_orders(order)
+                if not success:
+                    break
```

Comment:
Why not `return False`?

Reply:
Not sure what should be preferred here, result is the same anyway. Changed.